docs/getting-started/index.md

```diff
@@ -2,9 +2,15 @@
 title: "Getting Started"
 ---
 
-This is the high level documentation for Substrate, a next generation blockchain development framework.
+Welcome to the Substrate Developer Hub. 
 
-You might be looking for the reference documentation for Substrate's Rust codebase. You can find those `rustdocs` in our [Reference Documentation](/rustdocs/).
+As a next generation blockchain framework, Substrate is designed to help majority of developers build decentralized application to improve their business value. 
```

Comment:
Where are you getting these statements from?
I am not saying they are bad, but I feel that this kind of voice should be coming from some of our buisness folks or upper management.

Reply:
These statements are from my understanding with Substrate, I agree with you we should get confirmation from business folks or management people. 
Do you have a person in your mind to help review these statements?

Reply:
We'll work on a more approachable introduction to Substrate. For the time being, let's stick to the facts and feel free to ping me when in doubt. In this instance, please leave this out.

Reply:
Sure, already updated to be consistent with the project README.